### PR TITLE
Update plugin, author information

### PR DESCRIPTION
**Pull request by robincornett**
#9

### CODE REVIEW

**Review by LisaCee — COMMENTED** [review]
I'm not sure if we should handle this here or if I should make a new issue, but I searched "Katz" looking for any sneaky places we might have missed and there are a few dead links throughout the code to his support site.

**Comment by robincornett** [comment]
Nice catch! I'm also wondering if we need to keep the `tmp` and `docs` directories--maybe a separate issue.

**Review by LisaCee — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
-  "name": "katzwebservices/EDD-Gravity-Forms-Checkout",
+  "name": "easydigitaldownloads/EDD-Gravity-Forms-Checkout",
```
Comment (ashleyfae) [inline composer.json:2]:
```suggestion
  "name": "easydigitaldownloads/edd-gravity-forms-checkout",
```

To match the spacing in our repo name.

**Review by ashleyfae — CHANGES_REQUESTED** [review]
One tiny change, but otherwise looks good.

**Review by ashleyfae — APPROVED** [review]

**Pull request merged**